### ISSUE TITLE
hide accessibility strings that are not visible in the sim

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/rosetta#68]
For accessibility strings that are not visible (read by the screen reader), they will have a `"visible": "false"` field.  Hide these in Rosetta.

``` js
{
  “concentrationScreenDescription”: {
     “value”: “blah blah blah”,
     “visible”: false
   },
   ...
}
```

**Comment by pixelzoom** [comment]
@aaronsamuel137 will handle Rosetta.

@jonathanolson will makes sure visible:false is set for proper strings.

**Comment by aaronsamuel137** [comment]
Upon reflection, it's a little awkward to have a boolean field that defaults to true. Checking if the field is present does the reverse of what you'd expect for a boolean field:

```js
  if ( string.visible ) {
    // do something if string is invisible
  }
  if ( !string.visible ) {
    // do something if string is visible
  }
```

And it gets even more complicated if there will be cases when ```"visible": true``` will be explicitly set.

Any thoughts on this? What about changing the field to "invisible" which defaults to false?

**Comment by aaronsamuel137** [comment]
@pixelzoom, @jonathanolson please comment, can you comment on this?

**Comment by pixelzoom** [comment]
How about this?

`var displayInRosetta = ( string.visible === undefined ) ? true : string.visible;`

**Comment by pixelzoom** [comment]
How about this?

`var displayInRosetta = ( stringEntry.visible === undefined ) ? true : stringEntry.visible;`

... where `stringEntry` is the object literal that a string key maps to.

**Comment by aaronsamuel137** [comment]
Yeah that works, just thought it would look a little cleaner the other way. No problem going with that solution though.

**Comment by pixelzoom** [comment]
Since we use property 'visible' extensively elsewhere, I think it's preferable to stick with that, and live with a bit of awkward code here.

**Comment by aaronsamuel137** [comment]
Sounds good. I've added this check to rosetta. Assigning to @pixelzoom for review.

**Comment by pixelzoom** [comment]
This should go to @jonathanolson next, to add the "visible" field to strings.json.  

I also have no idea how I'd test this, so perhaps best for @jonathanolson to also confirm the Rosetta changes.

**Comment by jonathanolson** [comment]
Not sure why this would be hidden in Rosetta. Presumably we'd want to include accessibility strings to be translated.

@jessegreenberg, can you comment?

**Comment by pixelzoom** [comment]
@jessegreenberg Do you agree that this is a duplicate of https://github.com/phetsims/rosetta/issues/195 + https://github.com/phetsims/rosetta/issues/193?  If so, please close.

**Comment by jessegreenberg** [comment]
That is right, closing.

**Comment by liam-mulhall** [comment]
We are no longer using this `visible` field to hide a11y strings. I will remove references to this in Rosetta.

**Comment by liammulh** [comment]
It's hard to tell from the various issues linked, but it looks like this support (having a `visible` field) was maybe never added? Eventually we decided to prepend a11y strings with `a11y.`, and Rosetta ignores those string keys.

I have to add support for this for https://github.com/phetsims/forces-and-motion-basics/issues/298.
